app/api/events.py

```diff
@@ -578,18 +578,15 @@ def before_update_object(self, event, data, view_kwargs):
             else:
                 event.deleted_at = data.get('deleted_at')
 
-        deleted_tickets = 0
-        for ticket in event.tickets:
-            if ticket.deleted_at:
-                deleted_tickets += 1
+        deleted_tickets = event.tickets.filter(Ticket.deleted_at != None).count()
```

Comment:
As I said, this is not actual code. It won't work

Reply:
@iamareebjamal This is. When we put `lazy=dynamic`, it allows us to query into relationships using this syntax. 
Reference: https://stackoverflow.com/questions/15218732/filter-on-sqlalchemy-relationship

Reply:
So, is lazy_dynamic set?

Reply:
@iamareebjamal Check all file changes.